src/lua/utils.c

```diff
@@ -285,6 +286,19 @@ luaL_setcdatagc(struct lua_State *L, int idx)
 	lua_pop(L, 1);
 }
 
+int
+luaT_setfuncs(struct lua_State *L, const struct luaL_Reg *funcs)
+{
+	assert(lua_gettop(L) >= 1);
+	assert(lua_istable(L, -1));
+
+	for (const struct luaL_Reg *f = funcs; f->name != NULL; ++f) {
```

Comment:
Added `assert(funcs != NULL);`.

Reply:
Neat, thanks!

Reply:
(Irrelevant now, because `luaT_setfuncs()` was removed after a discussion in [this](https://github.com/tarantool/tarantool/pull/8173#discussion_r1083928591) thread.)